fuzzbench/jobs.py

```diff
@@ -19,14 +19,19 @@
 BASE_TAG = 'gcr.io/fuzzbench'
 
 
-def build_image(name: str):
+def build_image(tag: str, context: str, dockerfile=None, buildargs=None):
     """Builds a Docker image and returns whether it succeeds."""
-    image_tag = os.path.join(BASE_TAG, name)
+    image_tag = os.path.join(BASE_TAG, tag)
     subprocess.run(['docker', 'pull', image_tag], check=True)
-    subprocess.run(
-        ['docker', 'build', '--tag', image_tag,
-         os.path.join('docker', name)],
-        check=True)
+    command = ['docker', 'build', '--tag', image_tag, context]
+    cpu_options = ['--cpu-period', '100000', '--cpu-quota', '100000']
```

Comment:
As there is no `--cpus` option for `docker build`, we use this way, which is equivalent to `--cpus=1`.
refer: https://docs.docker.com/config/containers/resource_constraints/#configure-the-default-cfs-scheduler

Reply:
Looks good, thanks!